CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/CVKVisualProcessor.java

```diff
@@ -529,21 +565,23 @@ protected final VisualChangeProcessor getChangeProcessor(final VisualProperty pr
                 };      
             case EXTERNAL_CHANGE:
             default:
+                shouldRender = false;
```

Comment:
I'm not sure what the case for an EXTERNAL_CHANGE is?

Reply:
Me neither.